data/tooltips.txt

```diff
@@ -164,6 +164,15 @@ tip "hull heat multiplier:"
 tip "hull repair rate:"
 	`Amount of hull strength that can be repaired per second. Repairs usually consume energy and may generate heat as well.`
 
+tip "absolute threshold:"
+	`The remaining hull strength at which the ship becomes disabled, unaltered by any other attributes.`
```

Comment:
Might want to clarify the description for "absolute threshold." It could be interpreted as indicating the base value before it is altered.

`The remaining hull strength at which the ship becomes disabled. This value overrides any effects that change the disabled threshold.`

Reply:
Is that not already clear in the "unaltered by any other attributes" part?

Reply:
Not really. Saying that this is the value unaltered just means that *this* value hasn't been altered. It doesn't specify whether or not any other attributes will modify it afterwards.